glide-core/redis-rs/redis/src/cluster_async/connections_container.rs

```diff
@@ -217,14 +230,59 @@ where
                 let _ = slot_map_value.latest_used_replica.compare_exchange_weak(
                     initial_index,
                     index,
-                    std::sync::atomic::Ordering::Relaxed,
-                    std::sync::atomic::Ordering::Relaxed,
+                    Ordering::Relaxed,
+                    Ordering::Relaxed,
                 );
                 return Some(connection);
             }
         }
     }
 
+    pub(crate) fn round_robin_read_from_az_awareness_replica(
```

Comment:
Please, add comments on top of all new methods that you added or modified with new parameters or changed the return value